commands.js

```diff
@@ -376,6 +421,7 @@ var commands = exports.commands = {
 	officialchatroom: 'officialroom',
 	officialroom: function (target, room, user) {
 		if (!this.can('makeroom')) return;
+		if (room.isPersonal) return this.errorReply("Personal rooms configuration can't be changed.");
```

Comment:
This requires the `makeroom` permission anyway, so I'm not sure this is necessary.